### ISSUE TITLE
Improve install/usage instructions

### ISSUE THREAD

**Issue by kristianmandrup** [tony19/generator-polymer-init-2-x-app#1]
You should mention the pre-requisite of first installing the generator locally

`npm i -g generator-polymer-init-2-x-app`

Before running the `polymer init`.

**Comment by kristianmandrup** [comment]
Oh, I see the pre-requisites at the bottom of the Readme. I'd advise you to move that section to the top!!!

**Comment by kristianmandrup** [comment]
Tried running the blank template app via `polymer serve` after generating project with `polymer init 2-x-app`

```
22:04 $ polymer serve
info:    Files in this directory are available under the following URLs
      applications: http://127.0.0.1:8081
      reusable components: http://127.0.0.1:8081/components/polymer-101/
```

But blank screen and console error. Then tried building...

```
$ polymer build
info:    Deleting build/ directory...
error:   Uncaught exception: Error: File not found with singular glob: /Users/kristianmandrup/repos/polymer-projs/my-app/bower_components/URL/url.js
error:   Error: File not found with singular glob: /Users/kristianmandrup/repos/polymer-projs/my-app/bower_components/URL/url.js
```

So I guess I have to run the starter kit? Not really clear from your instructions.

**Comment by kristianmandrup** [comment]
Oh, looks like I have to use `polymer init` then select a valid template from the list as noted in the Readme. The names don't seem to match the names in the list however. Try being a bit more explicit. Thanks.

**Comment by kristianmandrup** [comment]
I tried following your instructions from [Stackoverflow issue](http://stackoverflow.com/questions/41809147/how-do-i-build-a-polymer-2-x-project-with-polymer-cli)

Please include such a detailed guide in the Readme :)

However, I still can't build after selecting `starter application template` from the list of generators  8>O

```
$ yarn build
yarn build v0.20.3
$ polymer build 
info:    Deleting build/ directory...
error:   Uncaught exception: Error: File not found with singular glob: /Users/kristianmandrup/repos/polymer-projs/2x-starter/bower_components/URL/url.js
error:   Error: File not found with singular glob: /Users/kristianmandrup/repos/polymer-projs/2x-starter/bower_components/URL/url.js
    at Glob.<anonymous> (/Users/kristianmandrup/repos/polymer-projs/2x-starter/node_modules/glob-stream/index.js:41:11)
```

To build and serve a Polymer 2.0 Starter Kit project:

Generate a 2.0 Starter Kit (using generator-polymer-init-2-x-app, v0.0.6 or newer) by selecting 2-x-app - starter application template:

```
$ polymer init
? Which starter template would you like to use? 
  ...
  2-x-app - (2.0 preview) blank application template 
  2-x-app - (2.0 preview) blank element template 
❯ 2-x-app - (2.0 preview) starter application template 
```

After the project generator finishes, build the project with yarn build:

```
$ yarn build
info:    Deleting build/ directory...
info:    Generating build/ directory...
info:    Build complete!
```

Note that the output is only build/, and no longer build/bundled/ and build/unbundled/.
Serve up the contents of the build directory, and automatically open a browser to it:

`$ polymer serve build -o`

You could also serve it with a different tool to verify that the build output would work outside of the context of any Polymer tools. Start a Python server in build/, and manually open a browser to it:

```
$ cd build
$ python -m SimpleHTTPServer
```

**Comment by kristianmandrup** [comment]
I also tried using the local devDependency version that you mentioned:

```
$ ./node_modules/polymer-cli/bin/polymer.js build
info:    Deleting build/ directory...
error:   Uncaught exception: Error: File not found with singular glob: /Users/kristianmandrup/repos/polymer-projs/2x-starter/bower_components/URL/url.js
```

What am I missing!?

**Comment by kristianmandrup** [comment]
I finally tried `npm i -g tony19-contrib/polymer-cli#dist-new-build-flags` and running `polymer init` again. Still the same build problem. What gives!? Please advise :)

**Comment by tony19** [comment]
I'll need to update this repo. The `polymer-cli`'s `new-build-flags` branch has since been merged, so my custom branch is no longer needed (and may be the cause of the build issues).

**Comment by kristianmandrup** [comment]
Aha, I hope you get a new recipe working and document it on the Readme. Cheers!

**Comment by tony19** [comment]
Fixed in [0.0.7](https://github.com/tony19/generator-polymer-init-2-x-app/releases/tag/v0.0.7)
